src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -22,9 +22,15 @@ export default class Tracks extends React.Component<TracksPropTypes, {}> {
     componentDidMount() {
 
         // --- construct params ---
-        let uniqCnasampleIds = _.uniq(_.map(this.props.cnaSegments, 'sampleId'));
-        let uniqMutSampleIds = _.uniq(_.map(this.props.mutations, 'sampleId'));
-        var config = tracksHelper.GenomicOverviewConfig(uniqCnasampleIds.length + uniqMutSampleIds.length, this.props.width);
+        // <old way> only show a track if theres data
+        //let uniqCnasampleIds = _.uniq(_.map(this.props.cnaSegments, 'sampleId'));
+        //let uniqMutSampleIds = _.uniq(_.map(this.props.mutations, 'sampleId'));
+        //var config = tracksHelper.GenomicOverviewConfig(uniqCnasampleIds.length + uniqMutSampleIds.length, this.props.width);
+        //</old way>
```

Comment:
Should we still keep the old way as an option that can be switched on/off via component props? Might be cleaner than just commenting out.